doc/devel/contrib/development.md

```diff
@@ -52,9 +52,9 @@ cmake --build build-clang
 ## Run the Test Suite:
 For running the test suite [ctest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) is used.
 Before one can run the test suite the `task_executable` must be built.
-After that also the `build_tests` target must be build, which can be done over:
+After that also the `test_runner` target must be build, which can be done over:
 ```sh
-cmake --build build --target build_tests
```

Comment:
It'd be nice if we could keep `build_tests` as an over-arching "build everything needed for tests" target, but it seems that [add_custom_target](https://cmake.org/cmake/help/latest/command/add_custom_target.html) makes a target that is _always considered out of date_ and thus would build on every CMake run -- that seems wasteful. Ideas?